projects/wvr-elements/src/lib/wvr-nav-list/wvr-nav-list.component.ts

```diff
@@ -1,22 +1,19 @@
 import { Component, Input } from '@angular/core';
 import { Alignment } from '../shared/alignment.enum';
-import { wvrThemeHostBindings, wvrThemeInputs } from '../shared/wvr-styles';
-
+/**
+ * The WvrNavList Component contains the aligned and vertical properties that describe the positioning of the ul list.
+ */
 @Component({
   selector: 'wvr-nav-list-element',
   templateUrl: './wvr-nav-list.component.html',
-  styleUrls: ['./wvr-nav-list.component.scss'],
-  inputs: [
-    ...wvrThemeInputs
-  ],
-  host: {
-    ...wvrThemeHostBindings
-  }
+  styleUrls: ['./wvr-nav-list.component.scss']
 })
 export class WvrNavListComponent {
 
+  /** The aligned property describing the positioning of the list elements. */
   @Input() aligned = Alignment.LEFT;
 
-  @Input() vertical = false;
+  /** This boolean vertical property describing if the list can be displayed horizontally or vertically. */
```

Comment:
Toggles the display of the list horizontally or vertically.

Reply:
Applied correction